apps/hash-frontend/src/pages/_app.page/error-fallback.tsx

```diff
@@ -0,0 +1,101 @@
+import { Box, Collapse, Tooltip, Typography } from "@mui/material";
+import { FallbackRender } from "@sentry/react";
+import { useState } from "react";
+
+import { Button } from "../../shared/ui/button";
+import { Link } from "../../shared/ui/link";
+
+const CopyableMonospace = ({ text }: { text: string }) => {
+  const [tooltipTitle, setTooltipTitle] = useState("Copy to clipboard");
+
+  return (
+    <Box>
+      <Tooltip title={tooltipTitle}>
+        <Button
+          onClick={async () => {
+            try {
+              await navigator.clipboard.writeText(text);
+              setTooltipTitle("Copied!");
+            } catch {
+              setTooltipTitle("Not allowed to copy to clipboard");
+            } finally {
+              setTimeout(() => setTooltipTitle("Copy to clipboard"), 3_000);
+            }
+          }}
+          variant="tertiary"
+          sx={({ palette }) => ({
+            background: palette.gray[20],
+            border: `1px solid ${palette.gray[50]}`,
+            color: palette.gray[90],
+            fontFamily: "monospace",
+            minHeight: 0,
+            py: 0.5,
+            px: 1.5,
+            "&:hover": {
+              background: palette.gray[40],
+            },
+          })}
+        >
+          {text}
+        </Button>
+      </Tooltip>
+    </Box>
+  );
+};
+
+export const ErrorFallback: FallbackRender = ({
+  error,
+  eventId,
+  resetError,
+}) => {
+  const [showMessage, setShowMessage] = useState(false);
+
+  return (
+    <Box
+      sx={({ palette }) => ({
+        background: palette.common.white,
+        margin: "0 auto",
+        padding: 6,
+        border: `1px solid ${palette.gray[20]}`,
+        borderRadius: 2,
+      })}
+    >
+      <Typography variant="h3" mb={2}>
+        Something went wrong...
+      </Typography>
+      <Typography mb={2}>
+        We've been notified and will investigate — please click the button below
+        to reset the page to its initial state.
+      </Typography>
+      <Box sx={{ textAlign: "center" }}>
+        <Button onClick={resetError} size="small">
+          Reset the page
+        </Button>
+      </Box>
+      <Typography mt={2}>
+        You can also <Link href="/contact">contact us</Link>. If you do, please
```

Comment:
We don't have a `/contact` page, is this in anticipation of having moved from `app.hash.ai` to `hash.ai`?

Reply:
Oops, nope – I've updated the link in https://github.com/hashintel/hash/pull/3368/commits/88dfc66eab15fa87567a80b1d1c0abff46db8396, thank you